assets/src/sass/blocks/_mailchimp-subscribe.scss

```diff
@@ -1,92 +1,109 @@
-.mailchimp-subscribe {
-	background: $background-color-framed;
-	border: 1px solid $border-color-framed;
-	border-radius: $border-radius-big;
-	font-family: $font-family-sans;
-	padding: rem(25);
-
-	h3 {
-		@include h3-sans;
+.block-editor-block-list__block.wp-block-mailchimp-mailchimp,
+.mc_container {
+	@include alignwide;
+	display: grid;
+	column-gap: rem(40);
+	padding: fluid(40, 80) 0;
+	row-gap: rem(16);
+
+	@media (min-width: #{ ($width-breakpoint-tablet) }) {
+		grid-template-columns: 1fr 1fr;
 	}
 
-	&__form:not(:only-child) {
-		@include global-margin-bottom;
+	&:not( :is( .alignfull .mc_container ) ) {
+		padding-left: rem(25);
+		padding-right: rem(25);
 	}
 
-	p {
+	h2, h3 {
 		font-family: $font-family-sans;
```

Comment:
Going to have an issue with `develop` at the moment because these variables are gone. Is this work that can be branched off the `feat-ux-refresh` branch?

Reply:
We are planning to release this work ahead of the UX refresh (if we get approval from legal). We'll need to make the changes needed to be compatible with the new theme work in a separate PR.